test/src/e2e_vm_tests/test_programs/should_pass/stdlib/u128_root_test/src/main.sw

```diff
@@ -4,26 +4,26 @@ use std::u128::*;
 
 fn main() -> bool {
 
-    let mut u_128: U128 = U128::from(0, 49);
-    let mut root_of_u_128 = u_128.sqrt();
-
-    assert(root_of_u_128 == U128::from(0, 7));
-
-    u_128 = U128::from(0, 25);
-    root_of_u_128 = u_128.sqrt();
-    assert(root_of_u_128 == U128::from(0, 5));
-
-    u_128 = U128::from(0, 81);
-    root_of_u_128 = u_128.sqrt();
-    assert(root_of_u_128 == U128::from(0, 9));
-
-    u_128 = U128::from(0, 144);
-    root_of_u_128 = u_128.sqrt();
-    assert(root_of_u_128 == U128::from(0, 12));
-
-    u_128 = U128::from(0, 1);
-    root_of_u_128 = u_128.sqrt();
-    assert(root_of_u_128 == U128::from(0, 1));
+    //let mut u_128: U128 = U128::from(0, 49);
```

Comment:
is this meant to be commented out for this PR?

Reply:
Yeah. The operation is now too expensive with the new gas schedule and the test was running out of gas. The implementation should probably be revamped. It's a new feature anyways (like 2 days ago).